js/src/product-feed/product-statistics/create-campaign-notice.js

```diff
@@ -0,0 +1,58 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { Notice } from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import AddPaidCampaignButton from '.~/components/paid-ads/add-paid-campaign-button';
+import useMCProductStatistics from '.~/hooks/useMCProductStatistics';
+import useAdsCampaigns from '.~/hooks/useAdsCampaigns';
+
+const CreateCampaignNotice = () => {
+	const { hasFinishedResolution: productsResolution, data: products } =
+		useMCProductStatistics();
+	const { loaded: campaignsLoaded, data: campaigns } = useAdsCampaigns();
+
+	const isLoading =
+		! productsResolution || products?.loading || ! campaignsLoaded;
+
+	const inValidData =
+		( productsResolution && ! products ) ||
+		( campaignsLoaded && ! campaigns );
+
+	if (
+		isLoading ||
+		inValidData ||
+		products?.statistics?.active === 0 ||
+		campaigns?.length > 0
+	) {
+		return null;
+	}
+
+	return (
+		<Notice className="gla-create-campaign-notice" isDismissible={ false }>
```

Comment:
@kt-12 Looks like we are not using this class name for any styling purposes. Can we remove it? Within your tests, you can use `hasText` to target that notice.